### PR TITLE
Adding Bicep template for App Configuration sample

### PR DESCRIPTION
**Pull request by willvelida**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by willvelida** [comment]
@bmoore-msft Please review :)

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          }
+          subnet: {
+            id: subnetRef
+          }
+        }
+      }
+    ]
+  }
+  dependsOn: [
+    virtualNetwork
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration/main.bicep:111]: remove dependsOn

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    ipConfigurations: [
+      {
+        name: 'ipconfig1'
+        properties: {
+          privateIPAllocationMethod: 'Dynamic'
+          publicIPAddress: {
+            id: publicIPAddress.id
+          }
+          subnet: {
+            id: subnetRef
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration/main.bicep:104]: use the bicep subnet ref

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+var nicName = 'myVMNic'
+var addressPrefix = '10.0.0.0/16'
+var subnetName = 'Subnet'
+var subnetPrefix = '10.0.0.0/24'
+var publicIPAddressName = 'myPublicIP'
+var vmName = 'SimpleWinVM'
+var virtualNetworkName = 'MyVNET'
+var subnetRef = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, subnetName)
+var appConfigRef = resourceId(appConfigStoreResourceGroup, 'Microsoft.AppConfiguration/configurationStores', appConfigStoreName)
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration/main.bicep:40]: create a symbolic name for this using the ```existing``` syntax

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    osProfile: {
+      computerName: vmName
+      adminUsername: adminUsername
+      adminPassword: adminPassword
+    }
+    storageProfile: {
+      imageReference: {
+        publisher: 'MicrosoftWindowsServer'
+        offer: 'WindowsServer'
+        sku: listKeyValue(appConfigRef, '2019-10-01', windowsOSVersionParameters).value
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration/main.bicep:131]: use the symbolic reference for getting the keys (here and below)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep (hunk 1 above)
Comment (willvelida) [reply quickstarts/microsoft.appconfiguration/app-configuration/main.bicep]: Removed in latest commit

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep (hunk 2 above)
Comment (willvelida) [reply quickstarts/microsoft.appconfiguration/app-configuration/main.bicep]: Resolved in latest commit

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep (hunk 4 above)
Comment (willvelida) [reply quickstarts/microsoft.appconfiguration/app-configuration/main.bicep]: Resolved in latest commit

File: quickstarts/microsoft.appconfiguration/app-configuration/main.bicep (hunk 3 above)
Comment (willvelida) [reply quickstarts/microsoft.appconfiguration/app-configuration/main.bicep]: Resolved in latest commit

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the pipeline failure

**Pull request closed without merging**